src/globus_cli/parsing/param_types/notify_param.py

```diff
@@ -0,0 +1,63 @@
+import click
+
+
+class NotificationParamType(click.ParamType):
+    def get_metavar(self, param) -> str:
+        return "{on,off,succeeded,failed,inactive}"
+
+    def convert(self, value, param, ctx):
```

Comment:
This is missing type annotations.

Reply:
Good call! I'll apply that fix and the docstring change, noted below.